control_panel_api/serializers.py

```diff
@@ -24,6 +24,9 @@ class Meta:
 
 
 class AppSerializer(serializers.ModelSerializer):
+
+    iam_role_name = serializers.ReadOnlyField(source='aws_role_name')
```

Comment:
I would not switch around field names as it causes confusion (I just had to trace code changes to figure out what this field was), just name it the same or edit one of them to match.